docs/ui/components/PageHeader/PageTitleButtons.tsx

```diff
@@ -6,20 +6,32 @@ import { Tag03Icon } from '@expo/styleguide-icons/outline/Tag03Icon';
 import { useRouter } from 'next/compat/router';
 
 import { githubUrl } from '~/ui/components/Footer/utils';
+import { MarkdownActionsDropdown } from '~/ui/components/MarkdownActions/MarkdownActionsDropdown';
+import { shouldShowMarkdownActions } from '~/ui/components/MarkdownActions/paths';
 import { FOOTNOTE } from '~/ui/components/Text';
 
 import { SdkPackageButton } from './SdkPackageButton';
 
 type Props = {
   packageName?: string;
   sourceCodeUrl?: string;
+  showMarkdownActions?: boolean;
 };
 
-export function PageTitleButtons({ packageName, sourceCodeUrl }: Props) {
+export function PageTitleButtons({ packageName, sourceCodeUrl, showMarkdownActions }: Props) {
   const router = useRouter();
+  const showEditButton = !sourceCodeUrl && !packageName && router?.pathname;
```

Comment:
What do you think of changing "Edit this page" to just "Edit" so the two options go together better?

<img width="753" height="118" alt="Screenshot 2025-09-19 at 10 03 28" src="https://github.com/user-attachments/assets/4c119f4e-0146-4dff-9926-89544ce4f196" />
<img width="764" height="108" alt="Screenshot 2025-09-19 at 10 03 20" src="https://github.com/user-attachments/assets/54e39edd-acad-4603-be66-4a448f96f759" />